app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,9 @@
+class SessionsController < ApplicationController
+  def create
+    # binding.pry
+    if user = User.from_google_auth(request.env["omniauth.auth"])
+      session[:user_id] = user.id
+    end
+    redirect_to dashboard_path
+  end
+end
```

Comment:
Do we need some sort of 'else' here? What happens if the google authentication fails? Do we know what we get back from google in that case?

Reply:
Discussion with team - If google auth fails, they just wouldn't end up directing the user back to the site, so they would never hit the session controller/create method.